var/spack/repos/builtin/packages/intel/package.py

```diff
@@ -0,0 +1,113 @@
+from spack import *
+import os
+
+class Intel(Package):
+    """Intel Compilers.
+
+    Note: You will have to add add the downolad file to a
+    mirror so that Spack can find it. For instructions on how to set up a
+    mirror, see http://software.llnl.gov/spack/mirrors.html"""
+
+    homepage = "http://www.intel.com"
```

Comment:
I would use https://software.intel.com/en-us/intel-parallel-studio-xe as the homepage.